ghtt/assignment.py

```diff
@@ -273,9 +275,26 @@ def create_repos(ctx, source, yes, students=None, groups=None):
             has_projects=False,
         )
 
+        # dangerous because we use methods with _
+        default_branch = ghtt.config.get('default-branch', 'master')
+        if default_branch != 'master':
+            g_repo._master_branch = g_repo._makeStringAttribute(default_branch)
```

Comment:
Always update default branch; to reduce edge cases and remove complexity. I'm also not sure what `_makeStringAttribute` does but `edit()` seems to be the recommended way to change settings of a pyGithub object. (this will be persisted in GitHub).

```suggestion
        g_repo.edit(default_branch=default_branch)
```